src/modules/contactForm/pages/MyContacts/MyContacts.tsx

```diff
@@ -73,9 +74,9 @@ type Props = {
     history: any,
     user: any,
     classes: any,
-    contacts: any,
+    contacts: Contact[],
```

Comment:
Yeah I went through a lot of the contact stuff and locked the types down. I thought it would be much easier to deal with in the future if we get that working now.